crates/ark/src/variables/variable.rs

```diff
@@ -434,6 +485,12 @@ impl WorkspaceVariableDisplayType {
 }
 
 fn has_children(value: SEXP) -> bool {
+    match ArkGenerics::VariableHasChildren.try_dispatch(value, vec![]) {
+        Err(e) => log::error!("Error from 'ark_variable_has_children' method: {e}"),
```

Comment:
`positron_` or whatever we decide on

Reply:
Fixed in https://github.com/posit-dev/ark/pull/560/commits/fdee0f42f4727615f5f3b31a4aa41c061f849781